docker_ci/apt_installs.sh

```diff
@@ -30,7 +30,8 @@ apt-get install -y \
   flex \
   libboost-all-dev \
   emacs \
-  libgtest-dev
+  libgtest-dev \
+  sudo
```

Comment:
Why is this needed now?

Reply:
Mostly convenience, and it seemed too petty for its own issue/PR.  When I test images, I want to be able to switch seamlessly between `root` to install packages and the user `dev` to make sure that environment works without involving another shell by just doing `sudo -u dev /bin/bash`.  Basically, we need `dev` so that people can't do non-standard things, and it's also much easier to enable SSH for a non-root user for multi-container MPI runs.

Reply:
👍🏼